test_regression.py

```diff
@@ -0,0 +1,180 @@
+
+#%%
+import pytest
+from pathlib import Path
+import numpy as np
+import os
+
+from utils import extract_test_geometries, extract_scenario_parameters, execute_and_assert
+
+os.environ["ENDPOINT"] = "https://openeo.dataspace.copernicus.eu/"
+
+
+def test_apply_kernel(auth_connection, tmp_path):
+
+    # Define scenario parameters
+    # Load scenario parameters
+    scenario_name = "apply_spatial_kernel"
+    params = extract_scenario_parameters(scenario_name)
+    
+    # Set up output directory and path
+    output_path = Path(tmp_path) / f"{scenario_name}.nc"
```

Comment:
FYI: tmp_path is already a `Path` object so you don't have to cast it again to that